proxy/envoy/policy.go

```diff
@@ -61,6 +61,7 @@ func insertMixerFilter(listeners []*Listener, instances []*model.ServiceInstance
 							"source.ip":  context.IPAddress,
 							"source.uid": context.UID,
 						},
+						QuotaName: "RequestCount",
```

Comment:
Sorry to come in late to the discussion...

The proxy must know which quota to request from the mixer by name, and how much to request. This is the only way that prefetching of quota can possibly work. In a fully-realized solution, the proxy will need to do a bit of processing to establish, for example, which method is being called in order to query the right quota bucket.

I really tried to make it so the proxy doesn't need to say which quota to get and how much, but it quickly becomes extremely complicated to handle caching/prefetching. So I abandoned that path.

In terms of calling for Quota on every incoming request, we are crafting a pretty nice prefetching algorithm which will generally hide any quota-related latency. Upon first request from a client, the proxy will issue a request for N tokens into the future and will use up those tokens. Before the tokens drop to 0, the proxy will make another request so that it hopefully always has a supply of tokens on hand. But in general, there will never be blocking on quota requests.